hi-ml/Makefile

```diff
@@ -55,8 +55,12 @@ mypy:
 	mypy --install-types --show-error-codes --non-interactive --package health_ml
 	mypy --install-types --show-error-codes --non-interactive --package testhiml
 
+# run black check
+black:
+	find . -name '*.py' -exec black --line-length=120 --skip-string-normalization --check --diff {} +
```

Comment:
Generally, I think it would be better to configure black via a config file, so that it use used consistently also if we invoke it via vscode (file `pyproject.toml` for example, 
```
[tool.black]
line-length = 120
target-version = ['py39']
```